dev-db/sqlitebrowser/sqlitebrowser-3.9.1.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="A database browser for SQLite"
+DESCRIPTION="DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite. \
+It is for users and developers wanting to create databases, search, and edit data. \
+It uses a familiar spreadsheet-like interface, and you don't need to learn complicated\ SQL commands.\
+HOMEPAGE="http://sqlitebrowser.org/"
+COPYRIGHT="COPYRIGHT 2012-2014 Martin Kleusberg"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/sqlitebrowser/sqlitebrowser/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d0d2e06a69927ba1d0b955f3261ce70c61befc5bd5ddaa06752dae8bb4219ed8"
+SOURCE_FILENAME="sqlitebrowser-$portVersion.tar.gz"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+    sqlitebrowser = $portVersion
+    libs:antlr-2.7.7
+    libs:qcustomplot-source
+    libs:qhexedit
+    libs:qscintilla
+    "
+REQUIRES="
+        haiku >= $haikuVersion
+        "
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsqlite3
+	devel:libqt5
+	"
+
+BUILD_PREREQUIRES="
+	haiku_devel
+        "
+
+PROVIDES_devel="
```

Comment:
Oops, we don't want any PROVIDES_devel, especially with items that are provided by another recipe!!
Well, I guess this was a copy/paste accident ;-)